README.md

```diff
@@ -41,3 +41,20 @@ If you want to deploy your own HomeApp which supports Google Login, make sure yo
 1. Click "Google".  
 1. Enter email address.  
 1. Click "Register".  
+
+# Using docker to serve
```

Comment:
Certain files are modified, therefore, deploy procedure are changed for those who are not using the docker version, right?

Reply:
Yes.
You can use a script `build.sh` to change variables in html source. (The procedure is not written yet)

Reply:
Thank you!

Reply:
@yoh1496 Can you add the steps for not using docker?

Reply:
I added deploy with nginx. Could you review this?